src/Mirza/BusinessRegistry/Client/Servant.hs

```diff
@@ -45,6 +45,7 @@ revokePublicKey  :: BasicAuthData -> BRKeyId -> ClientM RevocationTime
 addLocation      :: BasicAuthData -> NewLocation -> ClientM LocationId
 getLocationByGLN :: BasicAuthData -> LocationEPC -> ClientM LocationResponse
 searchLocation   :: BasicAuthData -> Maybe GS1CompanyPrefix -> Maybe UTCTime -> ClientM [LocationResponse] 
+uxLocation       :: BasicAuthData -> [GS1CompanyPrefix] -> ClientM [BusinessAndLocationResponse]
```

Comment:
I think you also ought to add this to the export list of this module so that I can use it :slightly_smiling_face: